partition/utils_test.go

```diff
@@ -28,20 +28,15 @@ type UtilsTestSuite struct {
 
 var _ = Suite(&UtilsTestSuite{})
 
-func (s *UtilsTestSuite) SetUpTest(c *C) {
-}
-
-func (s *UtilsTestSuite) TestRunCommand(c *C) {
-	err := runCommandImpl("false")
+func (s *UtilsTestSuite) TestRunCommandWithStdout(c *C) {
+	_, err := runCommandWithStdoutImpl("false")
 	c.Assert(err, NotNil)
 
-	err = runCommandImpl("no-such-command")
+	_, err = runCommandWithStdoutImpl("no-such-command")
 	c.Assert(err, NotNil)
-}
 
-func (s *UtilsTestSuite) TestRunCommandWithStdout(c *C) {
-	runCommandWithStdout = runCommandWithStdoutImpl
-	output, err := runCommandWithStdout("sh", "-c", "printf 'foo\nbar'")
+	output, err := runCommandWithStdoutImpl("sh", "-c", "printf 'foo\nbar'")
 	c.Assert(err, IsNil)
 	c.Assert(output, DeepEquals, "foo\nbar")
+
```

Comment:
spurious line. I'm just doing a quick scan here so I have nothing more useful to say :)